distribution/docs/src/main/resources/content/_coreConcepts/metadata-standards.adoc

```diff
@@ -0,0 +1,25 @@
+:type: subCoreConcept
+:status: published
+:title: Metadata Standards
```

Comment:
I'm not sure it is worth breaking out "data" from "metadata". Why do it? I think practically, "data" ends up in the content store and "metadata" doesn't. Is that true in all cases? What when we cross the line from "data" to "metadata" and is it worth making the distinction?

Reply:
I think the purpose was to separate the metadata formats from the data formats. I believe two files in the same file format can still have metadata aligned to different standards. Whether this matters or not, I'm not sure.